azure-pipelines.yml

```diff
@@ -22,15 +22,15 @@ pr:
 jobs:
   - job: build
     displayName: Build & Test
-    timeoutInMinutes: 10
+    timeoutInMinutes: "10"
     pool:
       vmImage: 'ubuntu-latest'
     variables:
       PIP_CACHE_DIR: ".poetry"
     steps:
       - checkout: self
         path: s
-        fetchDepth: 0
+        fetchDepth: "0"
```

Comment:
Do these not work as numbers?

Reply:
VSCode complains at me, I think I have a linter for azure pipelines installed. Can switch them back to numbers if preferred, works for either.

Reply:
If it still works as strings, i have no strong feelings either way. Are consistent for the whole file?